### PR TITLE
Added Overview + Proofread of Functional Requirements

### PR DESCRIPTION
**Pull request by evanugarte**

### CODE REVIEW

File: project_requirements/Project Requirement.md [hunk 1]
```diff
@@ -50,7 +50,7 @@
 <br />
 
 # Project Description
-XXXXXXXX
+The goal of this project is to create a food ordering system for customers at a food court. The motivation of this project is to help streamline the process for customers to order food. The process of food ordering will be sped up as customers will be able to clearly select what they wish to order from photos, food names and prices. Stake holders of this project will include the owners and managers of food businesses, as they would be using the product to sell food. Additional stakeholders include the programmers who implement the application and project managers of the project. Benefits to users include an improved expereince to order food. This improved experience will come from menus being easier to read with large photos and text being presented on a screen. Additional contributions to the experience will include having existing payment information be stored in the system, so the user can checkout faster. In addition to the improved experience, if a line forms to order food, users will likely spend less time in line as the system can reduce the amount of time spend ordering food.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > Stake holders of this project will include the owners and managers of food businesses, as they would be using the product to sell food.

> Additional contributions to the experience will include having existing payment information be stored in the system, so the user can checkout faster.

> line forms to order food, users will likely spend less time

Spliced commas.

File: project_requirements/Project Requirement.md [hunk 2]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
+Our system is designed for customers of all ages and genders. The system is designed to target everyone ranging from those who wish to purchase food from a store to those who just want to merely browse the list of available items. Users can access the website from anywhere, as long as they have access to the internet and an appropriate device, such as a desktop computer or mobile phone. There are two types of general users for this StoreFront application: registered and unregistered users. The registered user can enjoy all of the features of the StoreFront application, which includes browsing, selecting multiple items to purchase to add them to shopping cart, save favorite items, modify payment information and update their personal account information.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > The system is designed to target everyone ranging from those who wish to purchase food from a store to those who just want to merely browse the list of available items.

Remove "ranging"

> Users can access the website from anywhere, as long as they have access to the internet and an appropriate device, such as a desktop computer or mobile phone.

Spliced commas.

File: project_requirements/Project Requirement.md [hunk 3]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
-- Login/sign in
-  - Users can login to the webpage by entering their username and password.
-  - The system will check to see if the given username and its associated password are the same with the information that are stored in the database. If it is, the system will allow the users to access their account. If it is not, the system will display an error message through the webpage to let the user know what is wrong.
+  - Users will be able to create their account by entering their email, password, and additional information such as name. For those who already have an account and try to sign-up for a new one, the website will display a message alerting the user that their email is already in use for an existing account.
+  - The system will check if a user's email and username already exists in the database. If email/username wasn't found, the system will write the new email and password to the database and display a message to the letting users know that they have successfully created an account. If an email exists or is invalid, the system will display an error message to let users know that their account was not able to be created.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > display a message to the letting users know 

to 

> display a message letting users know

or

> display a message to the users letting them know

File: project_requirements/Project Requirement.md [hunk 4]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
+- Login/Sign Up
+  - Users can login to the webpage by entering their email and password.
+  - The system will check to see if the given email and its associated password are the same with the information that are stored in the database. If it is, the system will allow the user to access their account. If it is not, the system will display an error message through the webpage to let the user know if the email or password is incorrect.
 - Browse items
-  - Users can check out all the items that are displaying on the webpage including the image of the item, its name, and price.
-  - The system will access the information in the database to display the popular items in a specific format on the webpage for the user to see when they first login to the page.
+  - Users will be ablie to view all of the items in the database through the StoreFront webpage. Each item will have an associated image, name and price.
+  - The system will access the information in the database for each item to display items in a specific format on the webpage for the user to see when they first login to the page.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > - The system will access the information in the database for each item to display items in a specific format on the webpage for the user to see when they first login to the page.

Reword or change into two sentences

File: project_requirements/Project Requirement.md [hunk 5]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
 - Search items
-  - Users can use the system's search engine to search for specific items such as by category, name, etc.
-  - The system will access the database, sort out specific items, and display it as the result of the user search. If the items user looking for does not exist or out of stock, display a notify message to let users know.
-- Select items/add items in the cart
-  - Users can select the items that they are interested and add to the cart to purchase later while continue to browse the shop.
-  - System will keep track of current items in the cart inside the database for later use and only remove when the user delete the items or they have ordered the item(s).
-- Delete items in the cart
+  - Users will be able to use system's search tab to search for specific items by item fields, such as category and name.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > category and name

to 

> category or name

File: project_requirements/Project Requirement.md [hunk 6]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
+  - After a search is entered, the system will access the database to find all appropriate items. The results of the query will be displayed to the user, and if the query returned a total of zero items, a message will be displayed to let users know.
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > The results of the query will be displayed to the user, and if the query returned a total of zero items, a message will be displayed to let users know.

to

>The results of the query will be displayed to the user. If the query returned a total of zero items, then a message will be displayed to let the user know.

File: project_requirements/Project Requirement.md [hunk 7]
```diff
@@ -76,40 +76,40 @@ XXXXXXXX
...
+- Deleting items in the cart
   - Users can remove any item that they no longer want from the their shopping cart.
-  - The System will access a particular relation in the database and delete the entity of the specify item(s) that the user wishes to remove.
-- Add new payment method
+  - The System will access a particular relation in the database and delete the entity of the item(s) that the user specified to remove.
+- Add new payment methods
   - Users will be able to add a new credit card as a method payment by entering card number, card type, holder name, CVV (card verification value), and card expiration date.
-  - System will check with the database to see if the card number already exist or not. If it is, let the user know that this payment method already exist through the website. If it not, store all this information in a secure way in the database
-- Delete payment method
+  - System will check with the database to see if the card number already exist or not. If so, the user will be notified that this payment method already exists. If it not, store all this information in a secure way in the database
```
Comment (aaron-warren) [inline project_requirements/Project Requirement.md]: > If it not,

>If it does not,

**Comment by evanugarte** [comment]
Pr updated

**Pull request merged**